<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGIBug</class>
 <widget class="QMainWindow" name="CGIBug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CGI Bugger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionOpenDB"/>
    <addaction name="actionStartScanning"/>
    <addaction name="actionClearConsole"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenDB">
   <property name="text">
    <string>Open DB</string>
   </property>
  </action>
  <action name="actionStartScanning">
   <property name="text">
    <string>Start scanning</string>
   </property>
  </action>
  <action name="actionClearConsole">
   <property name="text">
    <string>Clear console</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
